bin/fmfexporter

```diff
@@ -54,4 +54,4 @@ if __name__ == '__main__':
         sys.exit(0)
 
     # Submitting filtered test cases
-    adapter.submit_testcase_list(tc_list)
+    adapter.submit_testcase_list(tc_list, parse_import_response=True)
```

Comment:
As @fgiorgetti mentioned, this should be False. I forgot this. ALso I wanted to implement this as arg-option (for polartion args), but I forgot about it. Anyway, this option is closely tight to next PR /commit - about sending this URL into jira as test case work link item.